docker/fc34/basic-setup.sh

```diff
@@ -69,4 +69,5 @@ cat <<EOF > /etc/sysctl.d/hugepages.conf
 vm.nr_hugepages=2
 EOF
 
-rpm -U /opt/rpms/*.rpm
+#rpm -U /opt/rpms/*.rpm
+rpm -U --force /opt/rpms/*.rpm
```

Comment:
Fixed in https://github.com/Mellanox/mkt/pull/70/commits/c2f86ca588ff03c2628e72049ad32caa603459fd